src/cudamatrix/cu-vector.cc

```diff
@@ -42,7 +42,8 @@ namespace kaldi {
 
 
 template<typename Real>
-Real VecVec(const CuVectorBase<Real> &a,
+Real VecVec(cublasHandle_t handle,
```

Comment:
VecVec, like all "user-level functions", should not take the Handle argument.  Get it from the CuDevice.